docs/index.rst

```diff
@@ -0,0 +1,65 @@
+
+metagraph-numba Documentation
+=============================
+
+metagraph-numba is a plugin for `Metagraph`_ that enables algorithms to be
+just-in-time (JIT) compiled with Numba.  This package currently does not
```

Comment:
Since this is the first mention of Numba, it'd be nice if it were linked to https://numba.pydata.org/ .

Reply:
Good suggestion.  Added.